h2o-core/src/main/java/water/util/Log.java

```diff
@@ -288,7 +262,7 @@ private static void setLog4jProperties(String logDir, java.util.Properties p) th
 
     p.setProperty("log4j.appender.HTTPD",                       "org.apache.log4j.RollingFileAppender");
```

Comment:
Aren't all block same except `Rx` and name of log level and its index? We can do DRY here... ;-)